.ci.yaml

```diff
@@ -327,6 +327,25 @@ platform_properties:
         {
           "sdk_version": "16c5032a"
         }
+  mac_arm64_new_ios:
```

Comment:
@vashworth Feel free to suggest a better name here. Previously, we've been adding the device name in this string, but iOS can do that or something different.